chromium_src/net/DEPS

```diff
@@ -2,6 +2,7 @@ include_rules = [
   "+../../../../net/base",
   "+../../../../net/cookies",
   "+../../../../net/dns",
+  "+../../../../net/http",
```

Comment:
these were added in a big batch for expediency, but they should be as specific as possible for adding new DEPS so please add a new DEPS in net/http and add this here (with an additional `../`)

Reply:
Fixed in latest revision.